contribution/self-managed-k8s/setup.sh

```diff
@@ -21,7 +21,7 @@ export DEBIAN_FRONTEND=noninteractive
 echo 'debconf debconf/frontend select Noninteractive' | sudo debconf-set-selections
 
 # install dependencies and llvm--toolchain
-sudo apt-get -y install build-essential libelf-dev pkg-config net-tools linux-headers-generic
+sudo apt-get -y install build-essential libelf-dev pkg-config net-tools linux-headers-$(uname -r) linux-tools-$(uname -r)
```

Comment:
`linux-*-generic` package always points to the default kernel the OS comes with. If the user updates the kernel, then it still points to older kernel version.

Reply:
@achrefbensaad Updated `setup.h`. The `linux-tools` package in Ubuntu has `bpftool`.